WalletWasabi.Tests/RegressionTests/BackendTests.cs

```diff
@@ -244,6 +244,9 @@ public async Task StatusRequestTestAsync()
 				var blockchainController = (BlockchainController)RegTestFixture.BackendHost.Services.GetService(typeof(BlockchainController));
 				blockchainController.Cache.Remove($"{nameof(BlockchainController.GetStatusAsync)}");
 
+				// Set back the time to trigger timeout, subtract one more minute to be sure.
+				global.IndexBuilderService.LastFilterBuildTime = DateTimeOffset.Now - BlockchainController.FilterTimeout - TimeSpan.FromMinutes(1);
```

Comment:
It must be `UtcNow`. Also, are you sure that substracting one extra minute has no side-effects?

Reply:
Fixed UtcNow. Removed the one-minute subtraction - it is not required.